src/SFML/PCH.hpp

```diff
@@ -0,0 +1,62 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2007-2022 Laurent Gomila (laurent@sfml-dev.org)
+//
+// This software is provided 'as-is', without any express or implied warranty.
+// In no event will the authors be held liable for any damages arising from the use of this software.
+//
+// Permission is granted to anyone to use this software for any purpose,
+// including commercial applications, and to alter it and redistribute it freely,
+// subject to the following restrictions:
+//
+// 1. The origin of this software must not be misrepresented;
+//    you must not claim that you wrote the original software.
+//    If you use this software in a product, an acknowledgment
+//    in the product documentation would be appreciated but is not required.
+//
+// 2. Altered source versions must be plainly marked as such,
+//    and must not be misrepresented as being the original software.
+//
+// 3. This notice may not be removed or altered from any source distribution.
+//
+////////////////////////////////////////////////////////////
+
+#ifndef SFML_SFML_PCH_HPP
+#define SFML_SFML_PCH_HPP
+
+////////////////////////////////////////////////////////////
+// Precompiled Headers
+////////////////////////////////////////////////////////////
+
+#include <SFML/Config.hpp>
+
+#ifdef SFML_SYSTEM_WINDOWS
+
+#define UNICODE  1
+#define _UNICODE 1
```

Comment:
This cannot lead to issues somewhere down the line? Or should they be `#undef`ed after including the Windows headers?

Reply:
This matches the same definitions in the CMakeLists, which are never undefined. It should not cause any issue:

```cmake
if(SFML_OS_WINDOWS)
    set(PLATFORM_SRC
        ${SRCROOT}/Win32/CursorImpl.hpp
        ...
    )
    source_group("windows" FILES ${PLATFORM_SRC})

    # make sure that we use the Unicode version of the Win API functions
    add_definitions(-DUNICODE -D_UNICODE)
```

Reply:
@Bromeon let me know if you're happy with that before merging :)

Reply:
Ok, fine with me in that case! 🚀